app/src/main/res/layout/fragment_update_health_prof_page.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="@drawable/ggreen"
+    android:padding="16dp">
+
+    <TextView
+        android:id="@+id/textViewHealthProfTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Update Health Professional"
+        android:textSize="24sp"
+        android:textStyle="bold"
+        android:layout_marginBottom="24dp"/>
+
+    <EditText
+        android:id="@+id/editTextHealthProfName"
+        android:layout_width="383dp"
+        android:layout_height="57dp"
+        android:hint="Health Professional Name"
+        android:inputType="text" />
+
+    <TextView
+        android:id="@+id/health_professional_update"
+        android:layout_width="match_parent"
+        android:layout_height="107dp"
+        android:background="@android:color/white"
+        android:clickable="false"
+        android:focusable="false"
+        android:longClickable="false"
+        android:padding="16dp"
+        android:text="This is a non-editable text"
+        android:textColor="@android:color/black"
+        android:textSize="16sp" />
+
+    <EditText
```

Comment:
Missing other values for health professional. Where does specialty come from? There's no specialty input in the create health prof page.

Reply:
done change and create health prof page